Added App Sketch and Description to M1 Proposal

Iskanu93 opened the pull request:
- Added `sketch.png` to `reports/img/`.  
- Provided a detailed description of the sketch, covering details on sidebar filters, dashboard components, and user interactions.

jessiezhang24 commented:
Looks great! But the img folder should be in the main directory instead of under `reports` folder

Iskanu93 commented:
> Looks great! But the img folder should be in the main directory instead of under `reports` folder

Moved the img/ folder to the main directory as per feedback and updated the proposal to reflect the correct image path.

jessiezhang24 reviewed (approved):
Great work!

jessiezhang24 commented:
closes #6
